packages/core/types/index.d.ts

```diff
@@ -82,7 +82,7 @@ export function ClassNames<Theme = any>(
 ): ReactElement<any>
 
 declare module 'react' {
-  interface DOMAttributes<T> {
+  interface HTMLAttributes<T> {
```

Comment:
Even with previous code, tests work well. Do you know why?
`HTMLAttributes` extends `DOMAttributes`, so `DOMAttributes` should work.

Reply:
![demo](https://user-images.githubusercontent.com/232559/49945184-94623080-feec-11e8-8a81-7722512b3bd9.gif)

Reply:
Does DOMAttr work with cli TS compiler? If so, it's something related with PHPStorm.
If not, could you tell me which TS version you use?

Reply:
Thanks for your prompt reply. The TS Compiler succeeds with `DOMAttributes`, it seems this is related to PHPStorm. I'll close this PR